mllib/src/main/scala/org/apache/spark/ml/stat/Correlation.scala

```diff
@@ -56,7 +56,7 @@ object Correlation {
    *  Here is how to access the correlation coefficient:
    *  {{{
    *    val data: Dataset[Vector] = ...
-   *    val Row(coeff: Matrix) = Statistics.corr(data, "value").head
+   *    val Row(coeff: Matrix) = Correlation.corr(data, "value").head
    *    // coeff now contains the Pearson correlation matrix.
    *  }}}
    *
```

Comment:
Also since we are here as well, there is a reference to input RDD up above in the docstring.